moped-etl/prefect/tasks/ecs.py

```diff
@@ -344,14 +344,15 @@ def create_task_definition(slug, api_endpoint):
         containerDefinitions=[
             {
                 "name": "graphql-engine",
-                "image": "hasura/graphql-engine:v2.7.0",
+                "image": "hasura/graphql-engine:v2.15.1",
                 # "image": "mendhak/http-https-echo:latest",
                 "cpu": 256,
                 "memory": 512,
                 "portMappings": [
                     {"containerPort": 8080, "hostPort": 8080, "protocol": "tcp"}
                 ],
                 "essential": True,
+                "healthCheck": { "command": ["CMD-SHELL", "curl -f http://localhost:8080/healthz || exit 1"]},
```

Comment:
the `-f` flag:

```
       -f, --fail
              (HTTP) Fail silently (no output at all) on server errors.
              This is mostly done to enable scripts etc to better deal
              with failed attempts. In normal cases when an HTTP server
              fails to deliver a document, it returns an HTML document
              stating so (which often also describes why and more). This
              flag will prevent curl from outputting that and return
              error 22.

              This method is not fail-safe and there are occasions where
              non-successful response codes will slip through,
              especially when authentication is involved (response codes
              401 and 407).

              See also --fail-with-body.
```

my new gripe is that it's really hard to quickly find docs for specific flags for command line tools. `curl` has soo many flags, and searching the `man` page for `-f` is not useful.  [here's](https://stackoverflow.com/questions/19198721/is-there-a-way-to-look-for-a-flag-in-a-man-page) a not helpful post about it.